dev/vite.config.ts

```diff
@@ -7,16 +7,19 @@ import {defineConfig} from "vite";
 // Create aliases for each package in the Perseus monorepo, so Vite knows
 // where to look when a file imports e.g. @khanacademy/perseus.
 const packageAliases = {};
-glob.sync(resolve("../packages/*/package.json")).forEach((packageJsonPath) => {
```

Comment:
I had to change this to use a base path to resolve relative to now that I'm importing it in the Storybook config.